packages/fxa-auth-server/test/remote/security_events.js

```diff
@@ -0,0 +1,153 @@
+    return token.id;
+  });
+}
+
+describe('remote securityEvents', () => {
+  let server;
+
+  before(function() {
+    this.timeout(15000);
+    config.securityHistory.ipProfiling.allowedRecency = 0;
+    config.signinConfirmation.skipForNewAccounts.enabled = false;
+    return TestServer.start(config).then(s => {
+      server = s;
+    });
+  });
+
+  it('returns securityEvents on creating and login into an account', () => {
+    const email = server.uniqueEmail();
+    const password = 'abcdef';
+    let client;
+
+    return Client.createAndVerify(
+      config.publicUrl,
+      email,
+      password,
+      server.mailbox
+    )
+      .then(x => {
+        client = x;
+        return client.login().then(() => {
+          return client.securityEvents();
+        });
+      })
+      .then(events => {
+        assert.equal(events.length, 2);
+        assert.equal(events[0].name, 'account.login');
+        assert.isBelow(events[0].createdAt, new Date().getTime());
+        assert.equal(events[0].verified, false);
+
+        assert.equal(events[1].name, 'account.create');
+        assert.isBelow(events[1].createdAt, new Date().getTime());
+        assert.equal(events[1].verified, true);
+      });
+  });
+
+  it('returns security events after password change, with unverified session', () => {
+    const email = server.uniqueEmail();
+    const password = 'oldPassword';
+    const newPassword = 'newPasssword';
+    let client;
+
+    return Client.createAndVerify(
+      config.publicUrl,
+      email,
+      password,
+      server.mailbox,
+      { keys: true }
+    )
+      .then(x => {
+        client = x;
+        return client.emailStatus();
+      })
+      .then(status => {
+        assert.equal(status.verified, true, 'account is verified');
+      })
+      .then(() => {
+        // Login from different location to created unverified session
+        return Client.login(config.publicUrl, email, password, { keys: true });
+      })
+      .then(c => {
+        client = c;
+      })
+      .then(() => {
+        // Ignore confirm login email
+        return server.mailbox.waitForEmail(email);
+      })
+      .then(() => {
+        return getSessionTokenId(client.sessionToken);
+      })
+      .then(sessionTokenId => {
+        return client.changePassword(newPassword, undefined, sessionTokenId);
+      })
+      .then(() => {
+        return client.securityEvents();
+      })
+      .then(events => {
+        assert.equal(events.length, 2);
+        assert.equal(events[0].name, 'account.login');
+        assert.isBelow(events[0].createdAt, new Date().getTime());
+        assert.equal(events[0].verified, false);
+
+        assert.equal(events[1].name, 'account.create');
+        assert.isBelow(events[1].createdAt, new Date().getTime());
+        assert.equal(events[1].verified, true);
+      });
+  });
+
+  it('returns security events after password change, with verified session', () => {
```

Comment:
We can remove this too. Instead try testing the password reset flow, ref https://github.com/mozilla/fxa/blob/b22d8cdeb47cf4dcd52aa90fc8e21f4eedd24725/packages/fxa-auth-server/test/remote/account_reset_tests.js#L157